frontend/app/locales/de.json

```diff
@@ -1,167 +1,167 @@
+  "authPanel.enable-comments": "Kommentarfunktion aktivieren",
+  "authPanel.enable-cookies": "Cookies zulassen, um die Kommentarfunktion zu verwenden",
+  "authPanel.hide-settings": "Einstellungen verstecken",
+  "authPanel.logged-as": "Du hast dich angemeldet als",
+  "authPanel.login": "Anmelden:",
+  "authPanel.logout": "Abmelden?",
+  "authPanel.or-provider": "oder",
+  "authPanel.other-provider": "Andere",
+  "authPanel.read-only": "Nur-Lesen",
+  "authPanel.request-to-delete-data": "Löschung meiner Daten anfragen",
+  "authPanel.show-settings": "Einstellungen anzeigen",
+  "blockingDuration.day": "Für einen Tag",
+  "blockingDuration.month": "Für einen Monat",
+  "blockingDuration.permanently": "Dauerhaft",
+  "blockingDuration.week": "Für eine Woche",
+  "comment.block": "Blockieren",
+  "comment.block-user": "Möchtest du den Benutzer {userName} wirklich {duration} blockieren?",
+  "comment.blocked-user": "Blockiert",
+  "comment.blocking-period": "Blockierdauer",
+  "comment.cancel": "Abbrechen",
+  "comment.controversy": "Kontroversen: {value}",
+  "comment.copied": "In die Zwischenablage kopiert!",
+  "comment.copy": "Kopieren",
+  "comment.delete": "Löschen",
+  "comment.delete-message": "Möchtest du diesen Kommentar wirklich löschen?",
+  "comment.deleted-comment": "Dieser Kommentar wurde soeben gelöscht",
+  "comment.deleted-user": "Gelöscht",
+  "comment.edit": "Bearbeiten",
+  "comment.expired-time": "Die Zeit für eine Bearbeitung ist abgelaufen.",
+  "comment.go-to-parent": "Gehe zum übergeordneten Kommentar",
+  "comment.hide": "Ausblenden",
+  "comment.hide-user-comment": "Möchtest du die Kommentare von Benutzer {userName} wirklich ausblenden?",
+  "comment.pin": "Anheften",
+  "comment.pin-comment": "Möchtest du diesen Kommentar wirklich oben anheften?",
+  "comment.reply": "Antworten",
+  "comment.time": "{day} um {time}",
+  "comment.toggle-verification": "Bestätigung umschalten",
+  "comment.unblock": "Freigeben",
+  "comment.unblock-user": "Möchtest du diesen Benutzer wirklich wieder freigeben?",
+  "comment.unpin": "Ablösen",
+  "comment.unpin-comment": "Möchtest du diesen Kommentar wirklich wieder ablösen?",
+  "comment.unverified-user": "Unbestätigter Benutzer",
+  "comment.unverify-user": "Möchtest du den Benutzer {userName} wirklich nicht mehr bestätigen?",
+  "comment.verified-user": "Bestätigter Benutzer",
+  "comment.verify-user": "Möchtest du den Benutzer {userName} wirklich bestätigen?",
+  "comment.vote-error": "Fehler beim Abstimmen: {voteErrorMessage}",
+  "commentForm.exceeded-size": "Die Datei {fileName} überschreitet das Größenlimit von {maxImageSize}",
+  "commentForm.input-placeholder": "Gib hier deinen Kommentar ein",
+  "commentForm.new-comment": "Neuer Kommentar",
+  "commentForm.notice-about-styling": "Hier kannst du <a>Markdown</a> verwenden",
+  "commentForm.preview": "Vorschau",
+  "commentForm.replay": "Antworten",
```

Comment:
replay should be **reply** instead

Reply:
Please change it manually. Don't worry, it already fixed in original dictionary and it will work well.